pyproject.toml

```diff
@@ -100,3 +102,11 @@ markers = [
     "run_on_testnet: marks test that will only run on testnet (when --net=testnet)",
     "run_on_devnet: marks test that will only run on devnet (when --net=devnet)"
 ]
+
+[tool.pyright]
+include = ["starknet_py/"]
+exclude = [
+    "**/__pycache__",
+    "starknet_py/tests",
+    "**/*_test.py",
```

Comment:
Why aren't we typechecking tests?

Reply:
There is no point in doing this currently.

Reply:
This should be mentioned in separate issue or in #390 then.

Reply:
I am not sure if we need to typecheck tests at all tbh

Reply:
I created an issue, but added a low priority https://github.com/software-mansion/starknet.py/issues/392